manifests/sysmgmt.yaml

```
@@ -198,7 +198,7 @@ spec:
     namespace: services
   - name: csm-config
     source: csm-algol60
-    version: 1.16.6
+    version: 1.16.7
```

Pull in changes for [CASMINST-6535](https://jira-pro.it.hpe.com:8443/browse/CASMINST-6535)
```suggestion
    version: 1.16.8
```